### ISSUE TITLE
Tonnika Sisters ignoring implied target rule

### ISSUE THREAD

**Issue by JediJer** [PlayersCommittee/gemp-swccg-public#517]
- [x] Did you search through the list of **open issues** _first_ to ensure that there is not already an report for this issue?

## **Describe the bug**
Tonnika Sisters were able to use their game text when no weapons or devices were present at the site. This was against a bot, so it is clearly an available option when it shouldn't be possible. This didn't cause any issues in the game as the attempts simply failed. However, it is not something that a person would likely do which may be why it hasn't been noticed

## **To Reproduce**
1. Deploy Tonnika Sisters to a site with no weapons or devices present
2. Next control phase, use Tonnika Sisters

## **Game Replay ID:**

https://gemp.starwarsccg.org/gemp-swccg/game.html?replayId=jedijer$rxcggxphxromy18y
DS turns 3 and 4, Tonnika Sisters were used when there was no LS character at the site.

## **Expected behavior**

 Implied target rule should prevent Tonnika Sisters from using their game text when there is nothing present to steal or destroy.

### **Desktop (please complete the following information):**
 - OS: Windows
 - Browser Edge
 - Version 98.0.1108.43

**Comment by EchoBaseTrooper** [comment]
Fixed the reported bug, noticed a few other issues with this card that still need to be fixed

**Comment by JediJer** [comment]
I honestly couldn't tell you if it was LS or DS that was able to choose, as I was playing against myself. It is possible that it was making LS choose

**Comment by BOTVHD** [comment]
> Fixed the reported bug, noticed a few other issues with this card that still need to be fixed

Aside from the wrong player getting a prompt ( #900 ), are there still some issues with Tonnika Sisters that you identified that still need to be fixed?

**Comment by Gergall** [comment]
For what it's worth, I have tested a bunch and while Tonnika Sisters inherently have some weird behaviors, they look to be correct.  Here are some "correct oddities":

-Tonnika Sisters is to be treated as a combo card, meaning they're never alone.
-Tonnika Sisters may destroy DS's own weapons/devices.
-Tonnika Sisters may draw a 2 and then choose "do nothing" because their game text says "may".

That's all quite weird but correct... only bug I see is where it lets LS choose steal vs destroy.

**Comment by EchoBaseTrooper** [comment]
Yes that's fine. I don't remember the other things I had noticed that needed to be fixed.
